package.json

```diff
@@ -14,6 +14,10 @@
     "bower": "^1.3.12",
     "c3": "^0.4.10",
     "css-loader": "^0.9.1",
+    "datatables-responsive": "^1.0.7",
```

Comment:
Refactoring this to use a read-only WaterButler provider will likely obviate these packages.